resources/schemas.json

```diff
@@ -72,11 +62,6 @@
             "type": "string",
             "description": "Any additional public info"
           },
-          "node": {
-            "type": "string",
-            "description": "The node info",
-            "example": "https://node01.logion.network"
-          },
           "logoUrl": {
```

Comment:
I think that the logo is now handled by the backend, so it could be removed as well.